fix: statusCode allow 999

riwu opened the pull request:
https://lodash.com/docs/#inRange `end` is exclusive, so it needs to be set to 1000 to allow status code 999 to be included (as per documented).

Closes https://github.com/cypress-io/cypress/issues/27567

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/cypress-io/cypress?pullRequest=27853) <br/>All committers have signed the CLA.

AtofStryker reviewed (approved):
Hi @riwu. Changes look good, but would you be able to update your branch by syncing the fork? It's almost a year out of date. This way CI can run correctly.

riwu committed:
fix: statusCode allow 999

https://lodash.com/docs/#inRange `end` is exclusive, so it needs to be set to 1000 to allow status code 999 to be included (as per documented).

astone123 committed:
Merge branch 'develop' into patch-1

Commit:
Merge branch 'develop' into patch-1

nagash77 commented:
build failures are unrelated to these changes.  merging.
